README.md

```diff
@@ -40,7 +40,7 @@
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-- [[Live Demo link](https://bhushan354.github.io/Superhero-Search-Project/)]
+- https://final-touch--merry-douhua-03f0b6.netlify.app/
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
```

Comment:
- [ ] As required by the last reviewer, Please add a proper project description here to the readme file.

![image](https://github.com/bhushan354/Superhero-Search-Project/assets/68796232/e05f153c-2207-48b7-bb03-87b6e0e1abb4)

[Note] Kindly submit this PR instead of the one you merged and leave the old PR mentioned as you did.